### PR TITLE
OHRM5X-885: PIM report validation improvement

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

File: src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss [hunk 1]
```diff
@@ -185,10 +185,10 @@
   @include oxd-respond-to('md') {
     height: 75vh;
   }
-  @media screen and (min-width: 1920px) {
+  @media screen and (min-width: 1800px) {
```
Comment (RajithaKumara) [inline src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss:188]: https://mui.com/material-ui/customization/breakpoints/#default-breakpoints
https://vuetifyjs.com/en/components/grids/#usage

File: src/client/src/core/util/validation/rules.ts [hunk 2]
```diff
@@ -361,7 +361,8 @@ export const validFileTypes = function (fileTypes: string[]) {
 export const validEmailFormat = function (value: string): boolean | string {
   return (
     !value ||
-    /^(([^<>()[\]\\.,;:\s@#\-"]+(\.[^<>()[\]\\.,;:\s@#\-"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(
+    // eslint-disable-next-line no-useless-escape
+    /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(
```
Comment (RajithaKumara) [inline src/client/src/core/util/validation/rules.ts:365]: seems better if we revert this;

https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/Validator/Constraints/EmailValidator.php#L29
https://www.w3.org/TR/2012/WD-html-markup-20121025/input.email.html#form.data.emailaddress_xref2

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss (hunk 1 above)
Comment (Super-Chama) [reply src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss]: Updated

File: src/client/src/core/util/validation/rules.ts (hunk 2 above)
Comment (Super-Chama) [reply src/client/src/core/util/validation/rules.ts]: Reverted

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**